packages/genprint/genprint.0.3/opam

```diff
@@ -26,6 +26,7 @@ depends: [
 ]
 
 url {
-  src: "https://github.com/progman1/genprintlib/archive/v0.3a.tar.gz"
-  checksum: "md5=d5b58cc0aa7e1dd745d94fcc405d9d27"
+  src: "https://github.com/progman1/genprintlib/archive/v0.3.tar.gz"
+
+  checksum: "md5=0f577477a139c27de8acbc9d5a878562"
```

Comment:
I'm sorry but we already talked about this in https://github.com/ocaml/opam-repository/pull/16199. Could you please not change previous releases? Your new `genprint.0.4` is fine though, why change the previous one?

Reply:
I updated the checksum but forgot to bump the src line release file number so opam file pointed to wrong tarball.

Reply:
the process is confusing - that was back when I tried to correct the 0.3 release thinking it was not even installable.
If I revert the details, is that safe to do?!

Reply:
yes